app/main.py

```diff
@@ -1,2 +1,91 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.hash_capacity: int = 8
+        self.hash_table: list = [None] * self.hash_capacity
+        self.iter_table: list = []
+        self.iter_no = 0
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        if self.hash_capacity / (self.__len__() + 1) <= 1.5:
+            self.resize()
+        hash_key = hash(key)
+        hash_index = hash_key % self.hash_capacity
+        while self.hash_table[hash_index] is not None:
+            hash_index: int = (hash_index + 1) % self.hash_capacity
+        self.hash_table[hash_index] = [key, hash_key, value]
+
+    def __getitem__(self, key: Any) -> Any:
+        hash_key = hash(key)
+        hash_index = hash_key % self.hash_capacity
+        try:
+            while self.hash_table[hash_index][1] != hash_key \
+                    and self.hash_table[hash_index][0] != key:
+                hash_index = (hash_index + 1) % self.hash_capacity
+            result_value = self.hash_table[hash_index][2]
+        except TypeError:
+            print("Not found that index!")
+            return
+        else:
+            return result_value
+
+    def __len__(self) -> int:
+        return sum([1 for value in self.hash_table if value])
```

Comment:
try to optimize sthis method

Reply:
I don't understand why checks of my solution have failed. Can you help me?